jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+by following the steps outlined in the
+<<approve-as-draft, editors' "Approve as Draft" section>>.
+When they are done, the Draft JEP will have an official JEP number and
+the sumbmission PR will have been merged to a matching folder and feature branch
+(i.e the
+`link:https://github.com/jenkinsci/jep/tree/jep-1/jep/1[jep/1]` folder in the
+`https://github.com/jenkinsci/jep/tree/jep-1/[jep-1]` branch).
+
+IMPORTANT: "Approval as Draft" is *not* the same as accepting the JEP.
+
+Editors are not the only ones who can approve as submission.
+Non-editor contributors who have "git push" privileges for the
+JEP repository footnoteref:[repo] may also approve submissions.
+When doing so, the developer must handle the tasks
+that would normally be taken care of by the JEP editors
+(see <<editor-responsibilities, JEP Editor Responsibilities & Workflow>>).
+This includes ensuring the initial version meets the expected standards
+for a Draft JEP.
+
+[[draft]]
+==== Refining a Draft JEP
+
+The version of a JEP that is approved as a Draft JEP
+is rarely the same as the final version that is reviewed and hopefully accepted.
+A Draft JEP often requires further refinement and expansion
+before it is suffiently complete and represents the consensus of the community.
+
+Standards Track JEPs consist of two parts, a design document
+and a reference implementation.
+At minimum, prototype implementation should be co-developed with the JEP,
+as ideas that sound good in principle sometimes turn out to be impractical
+when subjected to the test of implementation.
+
+A JEP's sponsor is responsible for collecting community feedback on a JEP
+before submitting it for review.
+Potential changes to a draft JEP may be discussed further on jenkinsci-dev@.
+However, long open-ended discussions are not recommended on mailing lists.
+Strategies to keep the discussion efficient include:
+
+* setting up a series of in-person, or video-conferencing sessions to
+  discuss the JEP with necessary stakeholders.
+* having the JEP sponsor accept private comments in the early design phases
+* setting up a wiki page, etc.
+* committing and reviewing small concrete changes via Pull Requests
+  rather than large sweeping changes
+
+JEP sponsors should use their discretion here.
+
+The JEP sponsor may also ask JEP editors for further feedback regarding the
+style and consistency of a JEP and its readiness for review by the BDFL.
+
+As updates are necessary, the JEP sponsor and other contributors
+should push commits to their fork of the JEP repository footnoteref:[repo],
+and submit pull requests targeting the JEP's feature branch.
+
+[[review]]
+==== JEP Review
+
+Once the sponsor believes a JEP is complete,
+they request the BDFL review the JEP for acceptance, usually via
+an email to the jenkinsci-dev@ mailing list.
+For a JEP that is predetermined to be acceptable
+(e.g., it is an obvious win as-is and/or its implementation has already been checked in)
```

Comment:
Huh?

Is that an indirect reference to 'informational' JEPs, or how could this happen?

Reply:
Yes, or any other reason the BDFL feels they can justify.